### DISCUSSION TITLE
How to change default ui/console 8080 to a different one.

### DISCUSSION THREAD

**Discussion by jadedevin13** [zitadel/zitadel#3867] [Q&A] [answered]
I was trying to port forward this app to a different port than 8080

<img width="958" alt="image" src="https://user-images.githubusercontent.com/6671182/175813101-b4a56bf9-13ea-437d-9a73-008194af8ff4.png">

Going to the console shows the following error.

<img width="1225" alt="image" src="https://user-images.githubusercontent.com/6671182/175813084-b38af803-fefe-4a0d-8ab3-a4f2d17842f1.png">

Then I've tried to port forward to port 8080 and it's now working correctly.

<img width="988" alt="image" src="https://user-images.githubusercontent.com/6671182/175813201-7a2a3906-bef6-48d4-a642-10e3d965c876.png">

**Comment by jadedevin13** [comment, marked as answer]
Saw that I can change redirect_urls from projections.apps_oidc_configs. Is there any way to make this easier upon setup like environment variables?

**Reply by fforootd** [reply]
FYI we just created #3874 to track the further improvements for this
